crates/tauri-utils/src/acl/build.rs

```diff
@@ -316,6 +316,25 @@ commands.deny = ["{command}"]
       .push(format!("deny-{slugified_command}"));
   }
 
+  use std::collections::HashSet;
+
+  let expected_files: HashSet<String> =
```

Comment:
We can push a list of generated files from this time near the `let out_path = path.join(format!("{command}.toml"));` above